.github/workflows/test.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby: [2.7, '3.0', 3.1]
+        ruby: [3.1, 3.2, 3.3, 3.4]
```

Comment:
Ruby 3.2, 3.3 and 3.4 is supported. 3.1 isn't anymore. Should we also remove it from the test workflows?